lib/razorpay/customer.rb

```diff
@@ -23,5 +23,17 @@ def self.edit(id, options = {})
     def self.all(options = {})
       request.all options
     end
+
+    def fetchTokens
+      self.class.request.get "#{id}/tokens" 
+    end
+
+    def fetchToken(tokenId)
+      self.class.request.get "#{id}/tokens/#{tokenId}" 
+    end
```

Comment:
Can you link the relevant doc for this? In desc it is mentioned as https://razorpay.com/docs/api/recurring-payments/upi/tokens/#21-fetch-token-by-payment-id

Reply:
@arjunmenon i recently check again the above link and i got that this doc is not belongs to this function and there is no doc for this . i got this function from php sdk so i had replicated in other sdk (node, python)

Reply:
<Internal> [Pls see](https://razorpay.slack.com/archives/C029ZL11PNK/p1648732059006459?thread_ts=1647927334.987909&cid=C029ZL11PNK) public docs are updated accordingly.